paket.lock

```diff
@@ -172,115 +172,96 @@ GITHUB
-    src/Compiler/Utilities/InternalCollections.fs (e1f735546a907b0a511b818f10829b1d75139021)
-    src/Compiler/Utilities/InternalCollections.fsi (e1f735546a907b0a511b818f10829b1d75139021)
-    src/Compiler/Utilities/lib.fs (e1f735546a907b0a511b818f10829b1d75139021)
-    src/Compiler/Utilities/lib.fsi (e1f735546a907b0a511b818f10829b1d75139021)
-    src/Compiler/Utilities/PathMap.fs (e1f735546a907b0a511b818f10829b1d75139021)
-    src/Compiler/Utilities/PathMap.fsi (e1f735546a907b0a511b818f10829b1d75139021)
-    src/Compiler/Utilities/QueueList.fs (e1f735546a907b0a511b818f10829b1d75139021)
-    src/Compiler/Utilities/QueueList.fsi (e1f735546a907b0a511b818f10829b1d75139021)
-    src/Compiler/Utilities/range.fs (e1f735546a907b0a511b818f10829b1d75139021)
-    src/Compiler/Utilities/range.fsi (e1f735546a907b0a511b818f10829b1d75139021)
-    src/Compiler/Utilities/rational.fs (e1f735546a907b0a511b818f10829b1d75139021)
-    src/Compiler/Utilities/rational.fsi (e1f735546a907b0a511b818f10829b1d75139021)
-    src/Compiler/Utilities/ResizeArray.fs (e1f735546a907b0a511b818f10829b1d75139021)
-    src/Compiler/Utilities/ResizeArray.fsi (e1f735546a907b0a511b818f10829b1d75139021)
-    src/Compiler/Utilities/RidHelpers.fs (e1f735546a907b0a511b818f10829b1d75139021)
-    src/Compiler/Utilities/sformat.fs (e1f735546a907b0a511b818f10829b1d75139021)
-    src/Compiler/Utilities/sformat.fsi (e1f735546a907b0a511b818f10829b1d75139021)
-    src/Compiler/Utilities/sr.fs (e1f735546a907b0a511b818f10829b1d75139021)
-    src/Compiler/Utilities/sr.fsi (e1f735546a907b0a511b818f10829b1d75139021)
-    src/Compiler/Utilities/TaggedCollections.fs (e1f735546a907b0a511b818f10829b1d75139021)
-    src/Compiler/Utilities/TaggedCollections.fsi (e1f735546a907b0a511b818f10829b1d75139021)
-    src/Compiler/Utilities/XmlAdapters.fs (e1f735546a907b0a511b818f10829b1d75139021)
-    src/Compiler/Utilities/XmlAdapters.fsi (e1f735546a907b0a511b818f10829b1d75139021)
-    src/Compiler/Utilities/zmap.fs (e1f735546a907b0a511b818f10829b1d75139021)
-    src/Compiler/Utilities/zmap.fsi (e1f735546a907b0a511b818f10829b1d75139021)
-    src/Compiler/Utilities/zset.fs (e1f735546a907b0a511b818f10829b1d75139021)
-    src/Compiler/Utilities/zset.fsi (e1f735546a907b0a511b818f10829b1d75139021)
-  remote: nojaf/FSharp.Formatting
-    FULLPROJECT (4203a0ab91b82a02d0a8dd54d345fbcf870ffed9)
-      CommandLineParser (>= 2.8 < 3.0)
-      FSharp.Compiler.Service (41.0.4)
-      FSharp.Core (>= 4.7)
-      FSharp.Data (>= 3.3 < 4.0)
-      FsUnit (>= 3.8 < 4.0)
-      Ionide.ProjInfo
-      Ionide.ProjInfo.Sln
-      Microsoft.Build.Framework
-      Microsoft.Build.Tasks.Core
-      Microsoft.Build.Utilities.Core
-      Microsoft.CodeAnalysis.CSharp (>= 3.6 < 4.0)
-      Microsoft.NET.Test.Sdk
-      Newtonsoft.Json
-      NUnit (>= 3.12 < 4.0)
-      NUnit3TestAdapter
-      Suave
-      System.Memory
+    src/Compiler/AbstractIL/il.fs (f8b56cb6db3c90348810263fd7a5d37dc5719e92)
```

Comment:
You are reverting the compiler to a different state here. A local build (`dotnet fake build`) could never have worked.